src/byte_patch_manager.cpp

```diff
@@ -83,6 +83,9 @@ namespace big
 
 		// Always send the special ability event
 		memory::byte_patch::make(g_pointers->m_gta.m_activate_special_ability_patch, std::to_array({0xB0, 0x01, 0xC3}))->apply();
+
+		// Crash the game if the game manages to send a report
+		memory::byte_patch::make((PVOID)g_pointers->m_gta.m_bonus_transaction, std::vector(0x60, 0x0))->apply();
```

Comment:
Couldn't we hook this to send a log at the least so we know this is happening or are you relying on the stack trace to convey this?

Reply:
It’s just a JMP to some encrypted code. I guess you could resolve the JMP and then inject there, call print stack trace, then kill the app.